src/components/forms/IamAccountSettingsForm.js

```diff
@@ -183,7 +183,7 @@ class IamAccountSettingsForm extends Component {
           />
           {/* session expiration */}
           <NumberInput
-            placeholder="900"
+            placeholder="(Optional) 900"
```

Comment:
let's move `(Optional)` here to label as well

Reply:
@jvallexm Do you mean (Optional) should only be in label and not in label and placeholder?  Having both seems like overkill. Wondering for the couple forms you said to move (optional) to label as well
![Screenshot 2023-08-15 at 10 26 57 AM](https://github.com/IBM/icse-react-assets/assets/80135387/992de846-76dd-44a3-bdc0-90e4a15431e3)

Reply:
I think we should follow the format that the IBM Cloud UI follows
<img width="840" alt="image" src="https://github.com/IBM/icse-react-assets/assets/39418171/b0e14469-6be6-4b8a-a71e-bdf7aeff1c18">